tests/admin_views/tests.py

```diff
@@ -864,6 +864,27 @@ def test_adminsite_display_site_url(self):
         self.assertEqual(response.context['site_url'], '/my-site-url/')
         self.assertContains(response, '<a href="/my-site-url/">View site</a>')
 
+    def test_submit_row_templatetag(self):
+        """
+        #13875 - submit_row templatetag should pass whole context
+        """
+        from django.contrib.admin.templatetags.admin_modify import submit_row
+        from django.contrib.auth.admin import UserAdmin
+        change_url = reverse('admin:auth_user_change', args=[self.superuser.pk])
+        factory = RequestFactory()
+        request = factory.get(change_url)
+        request.user = self.superuser
+        admin = UserAdmin(User, site)
+        extra_context = {'extra': True}
+        response = admin.change_view(request,
+                                     str(self.superuser.pk),
+                                     extra_context=extra_context)
+        template_context = submit_row(response.context_data)
```

Comment:
the `RequestFactory` instance allows me to retrieve a real `RequestContext` instance and pass it to the `submit_row` templatetag